test_conway.py

```diff
@@ -46,13 +46,7 @@ def test_there_can_only_be_corners(test_grid):
         (0, 0), (0, 9),
         (9, 0), (9, 9),
     }
-    for coordinates in conway.a_whole_new_world(test_grid):
-        if test_grid[coordinates]:
```

Comment:
this was the issue, we were testing *old* grid values

Reply:
Of course! Well done. 😀

Thank you for coming to pair with me, I learned a lot. See you at the next one!